endpoints/_includes/functions.php

```diff
@@ -746,6 +748,32 @@ function helplineSearch($latitude, $longitude)
     return json_decode(get($search_url));
 }
 
+function getBMLTServer($latitude, $longitude, $search_type)
+{
+    $helpline_search_radius = setting('helpline_search_radius');
+    if ($search_type == SearchType::VOLUNTEERS) {
+        if (setting('tomato_helpline_fallback')) {
+            $bmlt_search_endpoint = setting('tomato_url') . "/client_interface/json/?switcher=GetSearchResults&data_field_key=root_server_uri&sort_results_by_distance=1&long_val={LONGITUDE}&lat_val={LATITUDE}&geo_width=" . $helpline_search_radius;
+            $search_url = str_replace("{LONGITUDE}", $longitude, str_replace("{LATITUDE}", $latitude, $bmlt_search_endpoint));
+            $search_results = json_decode(get($search_url));
+            $pinged_BMLT_root = $search_results[0] -> root_server_uri;
+            $bmlt_root = $GLOBALS['bmlt_root_server']."/";
+            if ($pinged_BMLT_root != $bmlt_root) { // If the rootserver handling the phonecall is not the local rootserver, enable tomato routing - otherwise handle call routing locally.
+                $GLOBALS['settings_whitelist']['tomato_helpline_routing']['default'] = true;
+            }
+            return getHelplineBMLTRootServer();
+        } else {
+            return getHelplineBMLTRootServer();
```

Comment:
seems like both these conditions return the same thing, what's the difference?

Reply:
There isn’t - I originally had the return at the end outside of the if statement because of that, but you had said wrap it in and else statement for readability so I did.

The return doesn’t need to be inside the if because the only thing that portion of the if does is enable the setting tomato_helpline_routing to true if it is a volunteer search and tomato fallback is requested and the rootserver handling the routing is not the one the yap server is configured to.

I can move that back outside of the if statement where it originally was.

Reply:
I guess I didn't mean for it to wind up here.  But I also see that there is not always an exiting return happening, so it makes a little more sense now.